src/ExpressionEvaluator/Core/Source/Concord/Concord.csproj

```diff
@@ -52,7 +52,7 @@
           Outputs="$(OutDir)Phone\%(Identity).dll">
     <MakeDir Directories="$(OutDir)Phone" />
     <Exec Command="$(MSBuildFrameworkToolsPath)ilasm.exe /dll /quiet &quot;/output:$(OutDir)Phone\@(ConcordAssemblies).dll&quot; $(ConcordDir)@(ConcordAssemblies).Portable.il" />
-    <Exec Command="$(VSLOutDir)\FakeSign.exe &quot;$(OutDir)Phone\@(ConcordAssemblies).dll&quot;" />
+    <Exec Command="$(FakeSignToolPath) &quot;$(OutDir)Phone\@(ConcordAssemblies).dll&quot;" />
```

Comment:
I looked and found none.  If there are any though I'm happy to remove them as a part of this change.